developer/community-resources.md

```diff
@@ -0,0 +1,138 @@
+# Curated list of community-provided resources for Reaction
+
+This page is here to guide you through the Reaction ecosystem. As we've stated before, we'd not be here without our very engaging community. Lots of people are continuing to contribute to core development, but also supplying other members of the community with valuable plugins, tutorials or learning material. Here we'd like to introduce you to some of the awesome things we've seen out there.
```

Comment:
I think you want to say "engaged" rather than "engaging" though they are quite charming